app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,87 @@
+class TasksController < ApplicationController
+  def index
+    @tasks = Task.all
+  end
+  
+  def show
+    task_id_raw = params[:id]
+    task_id = task_id_raw.to_i
+    @task = Task.find_by(id: task_id)
```

Comment:
Just a nice-to-know: Actually, now that we have ActiveRecord up and running, ActiveRecord is very forgiving and we don't need to convert to integer... Aka, changing this code to this line still works:

```ruby
@task = Task.find_by(id: task_id_raw)
```